mirrord/operator/src/client.rs

```diff
@@ -243,9 +245,21 @@ impl OperatorApi {
             .await?
             .ok_or(OperatorApiError::InvalidTarget)?;
 
-        let target_to_connect = if config.feature.copy_target {
+        let target_to_connect = if config.feature.copy_target.enabled {
             let mut copy_progress = progress.subtask("copying target");
-            let copied = operator_api.copy_target(&metadata, raw_target).await?;
+
+            if config.feature.copy_target.scale_down {
+                let is_deployment = matches!(config.target.path, Some(Target::Deployment(..)));
+                if !is_deployment {
+                    progress.warning(
+                        "cannot scale down while copying target - target is not a deployment",
+                    )
```

Comment:
I think usually if the user specifies incompatible settings we exit with an error. So we should probably do the same here, unless @eyalb181 says otherwise. So I would also add a check to the config validation. But the check here is still necessary because if I'm not mistaken the target is not guaranteed to be known during config verification.

Reply:
I confirmed with Eyal, we want an error here. Updated